src/qt/forms/helpmessagedialog.ui

```diff
@@ -6,10 +6,13 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>780</width>
-    <height>400</height>
+    <width>602</width>
```

Comment:
The new configuration makes a better use of space - therefore the window doesn't need to open as big.

Reply:
The view's dimensions are now dynamic and resize based on help/about context.